### ISSUE TITLE
Inconsistent Parse Error on exported project

### ISSUE THREAD

**Issue by benbishopnz** [godotengine/godot#38424]
**Godot version:**
3.2.1.stable

**OS/device including version:**
Windows 10, MacOS Catalina

**Issue description:**
When I export my project, I consistently get a parse error when preloading certain scenes with my game. I previously exported without issue using an earlier version (although I'm not sure which.)
![image](https://user-images.githubusercontent.com/1631516/80898963-237c1c80-8d5e-11ea-89dd-591735335c9f.png)

The strange thing is, if I make a change to the file, delete the change and re-save it then I can export again and everything is fine... until I export the next time and the issues appears again :(

The file in question is rather simple, here is the contents:
```
extends "res://scripts/Player.gd"

func to_dict() -> Dictionary:
	var dict := .to_dict()
	dict.scene = "res://objects/Pignon.tscn"
	dict.ship = "Pignon"
	return dict

func _init():
	speed = 550
	set_max_health(4)

func _ready():
	if len(weapons) == 0:
		add_weapon("Charger", false)
		add_passive("Shield")
	emit_signal("switched_weapon", weapons[curr_weapon])

func _physics_process(delta):
	prev_vec = position - prev_pos
	prev_pos = position
```
I'm preloading this and another similar scene which extends the same script, but only this one gives issues.

This also happens with each of my level scenes (extremely frustrating) so I guess there is something in common with those.

**Steps to reproduce:**
Export my cursed project.

**Minimal reproduction project:**
This project has been upgraded starting from Godot 3.0.1 so possibly something went bad along the way...

**Comment by ThakeeNathees** [comment]
could you upload a Minimal reproduction project, the error log seems like your resource ("Pigon.tscn") found but invalid.

**Comment by benbishopnz** [comment]
Thanks for the comment. I was stripping down my project to try and create a minimal reproduction and I noticed the Pignon scene was using a script that was not in the folder I had thought it was in, so I moved it and then realised the .tscn file had a reference to a folder I must have renamed from capitalised to non-capitalised. I checked the level files, unlinked the scripts and reloaded then into the scenes and they had the same issue - I must have renamed the folders at some point after the last time I exported my project (and not via the Godot editor.)

I haven't confirmed this fixes the export yet, but I am 99% certain this was the cause. Not sure why re-saving the scripts before exporting was fixing the exported game though, that is a weird side-effect.

![image](https://user-images.githubusercontent.com/1631516/82614825-daacda80-9c1c-11ea-8dc0-f20c938c151d.png)
